ops/terraform/services/server/server-load/user-data.sh.tftpl

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+# add a timestamp to this scripts log output and redirect to both console and logfile
+exec > >(
+    while read line; do
+        echo "$(date +"%Y-%m-%d %H:%M:%S") - $${line}" | tee -a /var/log/user_data.log 2>&1
+    done
+)
+
+ansible localhost -m git -a 'repo=https://github.com/CMSgov/beneficiary-fhir-data.git dest=/beneficiary-fhir-data version=${git_repo_version} force=true'
+
+cd /beneficiary-fhir-data/apps/utils/locust_tests
+python3 -m pip install --upgrade pip --quiet
+python3 -m pip install -r requirements.txt --quiet
+PATH="/usr/local/bin:$${PATH}"
+chmod +x lambda/server-load/controller.sh
+
+# TODO: This IS NOT prodction ready until a non-privileged user runs load test logic; BFD-1883 et al
```

Comment:
Is this a subtask/story we need to write/execute?

Reply:
Yes, this will be addressed in BFD-1883.

Reply:
This will have to remain... as far as how it's planned and when/where it's done... I couldn't say. BFD-1783 has already accommodated so much. Maybe a little lazy to simply defer to the _next_ logical story, but I digress. This comment of not being production ready might also be a _little_ incendiary, but, it is difficult to defend running as the root user, amirite?

Reply:
As discussed with @malessi, we're aware of the issues here. We'll solve for this as part of BFD-1883.